nesta/packages/dq/nested.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+import itertools
+from collections import Counter
+##arrays
+def array_length(input):
+    '''array_length
+    Args:
+        input (:obj:`iter` of :obj:`list`): A sequence of list objects.
+        column (:obj:`list`): A list of column name strings.
+        type_ (:obj:`str`): A string of the type of column in question.
+
+    Returns:
+        array_length (:obj:`list`): An array of array lengths.
+    '''
+    series = pd.Series(input)
+    # array_length = dataframe[column].apply(lambda x: len(x) if type(x) == type_ else None).values
+    array_length = series.apply(lambda x: len(x) if type(x) == list else None)
+
+    return array_length
+
+def word_arrays(input):

```

Comment:
Looks like the code inside this function is duplicated a few times throughout the rest of the module.